clang/test/Driver/print-multi-selection-flags.c

```diff
@@ -1,79 +1,79 @@
-// RUN: %clang -print-multi-flags-experimental --target=aarch64-linux -fc++-abi=itanium -fsanitize=address | FileCheck --check-prefix=CHECK-LINUX %s
+// RUN: %clang -multi-lib-config=%S/Inputs/multilib/empty.yaml -print-multi-flags-experimental --target=aarch64-linux -fc++-abi=itanium -fsanitize=address | FileCheck --check-prefix=CHECK-LINUX %s
```

Comment:
Do you actually need to add this argument in this file? The file is only testing the flags and not the chosen final path, so the (lack of) YAML file should have no effect (`--sysroot=` isn't passed)

Reply:
This is needed since #105684 allows reporting errors from multilib.yaml for option combination there is no library for. This causes the tests to fail when building clang as part of https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm, which has a multilib.yaml which uses that feature. Adding the option here makes the tests independent of other parts of the toolchain it is built into.